### PR TITLE
Adding Serbian Language.

### PR DESCRIPTION
**Pull request by AndresMorelos**
Many Thanks to @milejancic

### CODE REVIEW

File: i18n/sr/settings.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    "requiredFields": "Обавезна поља",
+    "sound": "Звук",
+    "mute": "Нечујно",
+    "autoCheckUpdate": {
+      "name": "Аутоматска провера ажурирања",
+      "daily": "Сваки дан (Препоручено)",
+      "weekly": "Сваке недеље"
+    },
+    "currency": {
+      "placement": "Место за потпис",
```
Comment (milejancic) [inline i18n/sr/settings.json:27]: In context this should be:
Место за ознаку валуте

File: i18n/sr/messages.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "noContact": "Још увек немате ниједан контакт",
```
Comment (milejancic) [inline i18n/sr/messages.json:2]: I missed a space

"noContact": "Још увек немате ни један контакт",

**Review by milejancic — APPROVED** [review]
Looks great with couple of misses on my part which I marked in the comments.

**Comment by AndresMorelos** [comment]
@milejancic the typos were updated.

**Pull request merged**